docs/src/main/asciidoc/datasource.adoc

```diff
@@ -410,18 +394,19 @@ quarkus.datasource.inventory.jdbc.url=jdbc:h2:mem:inventory
 quarkus.datasource.inventory.jdbc.max-size=12
 ----
 
-Notice there is an extra bit in the key.
+Notice there is an extra section in the configuration key.
 The syntax is as follows: `quarkus.datasource.[optional name.][datasource property]`.
 
-NOTE: Even when only one database extension is installed, named databases need to specify at least one build time
-property so that Quarkus knows they exist. Generally this will be the `db-kind` property, although you can also
+NOTE: Even when only one database extension is installed, named databases need to specify at least one build-time property so that Quarkus can detect them.
+Generally, this is the `db-kind` property, but you can also
 specify Dev Services properties to create named datasources (covered later in this guide).
 
-=== Named Datasource Injection
+==== Named Datasource Injection
 
 When using multiple datasources, each `DataSource` also has the `io.quarkus.agroal.DataSource` qualifier with the name of the datasource as the value.
-Using the above properties to configure three different datasources, you can also inject each one as follows:
 
+. Using the above-mentoned properties to configure three different datasources, inject each one as follows:
```

Comment:
Again, is this really intended as a single-item list?

Reply:
All of these are numbered procedural steps. Sometimes, it is a shame that there is only one, but regarding the way tech docs are written in RH, we need to address them with a number (for single or several steps) or a star (to make it a single-item bullet list - for a single step)